[DOC] Kafka config example (#1751)
strimzi/strimzi-kafka-operator: documentation/book/con-kafka-listeners.adoc

* `tls`
* `external`

> When none of these properties are defined, the listener is disabled.

Reviewer: This sounds strange ... should it rather say

```suggestion
When the property for given listener is not defined, the listener is disabled.
```

> When the property for given listener is not defined, the listener is disabled.

Reviewer 2: It's a bit weird to talk about "the given listener" while simultaneously talking about that thing being absent.

```suggestion
Each listener will only be defined when the `listeners` object has the given property.
```
